src/examples/local_events_zkapp.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  Field,
+  state,
+  State,
+  method,
+  UInt64,
+  PrivateKey,
+  SmartContract,
+  Mina,
+  Party,
+  isReady,
+  Permissions,
+  DeployArgs,
+  UInt32,
+  Bool,
+  PublicKey,
+  Circuit,
+  Experimental,
+  CircuitValue,
+  prop,
+} from 'snarkyjs';
+
+const doProofs = false;
+
+await isReady;
+
+class Event extends CircuitValue {
+  @prop pub: PublicKey;
+  @prop value: Field;
+
+  constructor(pub: PublicKey, value: Field) {
```

Comment:
We had a discussion awhile back about adding a static `from` method that invokes the original constructor instead of overriding the constructor in examples to encourage this pattern. Here is the [discussion](https://github.com/o1-labs/snarkyjs/pull/269) for background and here is an [example](https://github.com/o1-labs/snarkyjs/pull/269/files#diff-4f28cac68e5aaf84bb21ddd42725b110f5ba1665c49ffe63164e6a9ad0c97165R23).

What you are doing works (passing constructor arguments to super) but we want to encourage the `.from `pattern to the community. @mitschabaude do you still think we should encourage this `.from` pattern in our examples?

Reply:
Uhm, to be honest, I'm not sure. I think it's fine either way. My original idea was that you'd have to do nothing and use the generic constructor. `.from` would've only be for the few cases where you'd want to transform the input arguments. However, by now I think a constructor like this, which is a no-op, is still nice to add, just so it becomes properly typed

Reply:
That works for me. I'm fine leaving it as is if we are in agreement.